lib/dmv_factory.rb

```diff
@@ -0,0 +1,25 @@
+class DmvFactory
+  attr_reader :office_list
+  def initialize
+    @office = nil
+  end
+
+  def create_offices_helper(api_data)
+    @office_list = api_data.map do |office|
+      @office = {
+       name: "#{office[:dmv_id]} #{office[:dmv_office]}", 
+       address: "#{office[:address_li]} #{office[:address__1]} #{office[:city]} 
+                 #{office[:state]} #{office[:zip]}", 
+       phone: office[:phone], 
+       services: office[:services_p]
+      }
+    end
+  end
+
+  def create_offices(api_data)
+    self.create_offices_helper(api_data)
+    offices = @office_list.map do |office|
+     facility = Facility.new(office)
+    end
+  end
```

Comment:
1. The use of `self` within methods is implied in Ruby. 
2. If you were to take my advice in the previous comment about doing away with the `@office_list` ivar, this might look something like
```
def create_offices(api_data)
  office_list = create_offices_helper(api_data)
  office_list.map do |office|
    Facility.new(office)
  end
end
```
Note that I removed the unused variables `offices` and `facility` as well.